src/components/UI/Buttons/IconButton.vue

```diff
@@ -2,11 +2,11 @@
   <button
     type="button"
     :class="buttonClass"
-    class="flex justify-center items-center transition ease-in duration-200 text-center text-base font-semibold focus:shadow-none focus:outline-none disabled:opacity-50 disabled:cursor-not-allowed"
+    class="flex justify-center items-center transition ease-in duration-200 text-center text-base font-semibold focus:shadow-none focus:outline-none disabled:opacity-50 disabled:cursor-not-allowed group"
```

Comment:
This is not imposing.
If want to do a group hover on an IconButton, there was no way to do that before this. Now I can do group hover from the parent where IconButton is being used. If I don't want to do group hover, I won't type `group-hover` in the parent component.

This was added because the settings button in Editor, I wanted the button to turn primary color and the settings cog icon to turn white on hover and I could only do that if I had grouped both the <button> and <SVG> defined inside IconButton.vue